lstchain/reco/dl0_to_dl1.py

```diff
@@ -306,15 +306,15 @@ def r0_to_dl1(input_filename=get_dataset_path('gamma_test_large.simtel.gz'),
                     # whenever GPS time is dumped to gps_time field.
 
                     if not is_simu:
-                        ucts_timestamp = event.lst.tel[telescope_id].evt.ucts_timestamp
+                        # ucts_timestamp = event.lst.tel[telescope_id].evt.ucts_timestamp
  
-                        if ucts_timestamp != 0:
-                            # UCTS timestamps are nanoseconds in UNIX timestamp format and must be given in seconds
-                            utc_time = Time(datetime.utcfromtimestamp(ucts_timestamp / 1e9))
-                        else:
-                            start_ntp = event.lst.tel[telescope_id].svc.date
-                            ntp_time = start_ntp + event.r0.tel[telescope_id].trigger_time
-                            utc_time = Time(datetime.utcfromtimestamp(ntp_time))
+                        # if ucts_timestamp != 0:
+                        #     # UCTS timestamps are nanoseconds in UNIX timestamp format and must be given in seconds
+                        #     utc_time = Time(datetime.utcfromtimestamp(ucts_timestamp / 1e9))
+                        # else:
+                        start_ntp = event.lst.tel[telescope_id].svc.date
```

Comment:
So, now it avoids UCTS time and uses TIB time. This is what you were using before you incorporated UCTS time, right? If so, it seems ok to me.

Reply:
Yes, this is how I was first taking time info

Reply:
Ok, it is then already tested. It should be fine then. Let's then wait for what @FrancaCassol's comments